test/gtest/common/test.cc

```diff
@@ -12,24 +12,252 @@
 #include <ucs/sys/sys.h>
 
 #include <memory>
+#include <mutex>
 
 namespace ucs {
+namespace log {
+
+/* helper structure to control all static variables in single place */
+static struct instance {
```

Comment:
is the refactoring of putting global vars in a class critical for entity storage change?

Reply:
AFAIR it was needed to have access to logging variables and functions from outside of `test_base` scope